metagraph/src/cli/config/config.cpp

```diff
@@ -442,6 +442,7 @@ Config::Config(int argc, char *argv[]) {
     if (identity != CONCATENATE
             && identity != STATS
             && identity != SERVER_QUERY
+            && !(identity == TRANSFORM_ANNOTATION && anno_type == Config::RowDiff)
             && !(identity == BUILD && complete)
             && !fnames.size())
         print_usage_and_exit = true;
```

Comment:
we should add a message here to explain that at least one file needs to be passed